Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityInputSystem.cs

```diff
@@ -745,15 +719,23 @@ public void RaiseFocusEnter(IMixedRealityPointer pointer, GameObject focusedObje
                 delegate (IMixedRealityFocusHandler handler, BaseEventData eventData)
                 {
                     var casted = ExecuteEvents.ValidateEventData<FocusEventData>(eventData);
-                    handler.OnFocusEnter(casted);
+                    if (handler.ReceiveAllFocusChanges)
+                    {
+                        handler.OnFocusEnter(casted);
+                    }
+                    else
+                    {
+                        if (casted.FocusProvider.IsOnlyFocusingPointer(casted.NewFocusedObject, casted.Pointer))
+                        handler.OnFocusEnter(casted);
```

Comment:
It's a little unclear without the braces, but I think the indentation is off here. Preferably, braces would be added here and on line 758 for readability.

Reply:
done